metaschema-cli/src/main/java/gov/nist/secauto/metaschema/cli/commands/AbstractConvertSubcommand.java

```diff
@@ -145,8 +145,10 @@ public void execute() throws CommandExecutionException {
             handleConversion(source, toFormat, writer, loader);
           }
         }
-      } catch (IOException | IllegalArgumentException ex) {
+      } catch (IllegalArgumentException ex) {
```

Comment:
Is there a reason we don't a step through exception catch on the generic Exception type for unexpected errors just to log them if no action taken?

Reply:
This is about passing on their messages to the CLI handler with the appropriate error code.